python/tk_multi_screeningroom/screeningroom.py

```diff
@@ -53,20 +56,32 @@ def _launch_rv(base_url, cmd, source=None, path_to_rv=None):
     if source:
         args.append(source)
 
+    combined_args = " " + " ".join(args)
     # We should encode the args so that no special characters exist in the command
-    encoded_args = (" " + " ".join(args)).encode("hex")
+    # Encode the string into binary so that we can convert the args to a hex
+    binary_str = six.ensure_binary(combined_args)
+    # Encode the binary into a hex
+    encoded_args = codecs.encode(binary_str, "hex")
+    # Now convert the binary back into a string which contains the hex representing the args.
+    hex_encoded_args_string = six.ensure_str(encoded_args)
 
     # If no path to RV was provided, us the SG site rvlink protocol to launch RV
     if not path_to_rv:
         # Encode the RV args. We'll use shotgun to redirect to the RV app via the rvlink
         # custom protocol
-        url = "%s/rvlink/baked/%s" % (base_url, encoded_args)
+        url = "%s/rvlink/baked/%s" % (base_url, hex_encoded_args_string)
         webbrowser.open(url)
     else:
-        url = "rvlink://baked/%s" % (encoded_args)
+        url = "rvlink://baked/%s" % (hex_encoded_args_string)
         cmdLine = " ".join(['"%s"' % path_to_rv, url])
         print("Running %s" % cmdLine)
         subprocess.Popen(cmdLine, shell=True)
```

Comment:
It's a bummer we're shelling out this way here with `shell=True` and a string command, but I also don't think we should worry about it as part of this work. Technically, what we're doing here is less secure than it should be, and generally not considered good Python practice.